### ISSUE TITLE
In Cyclical_Focal_Loss, what's the meaning of self.targets_classes?

### ISSUE THREAD

**Issue by whuhxb** [lnsmith54/CFL#3]
Hi, 

   In Cyclical_Focal_Loss, what's the meaning of self.targets_classes= = torch.zeros_like(inputs).scatter_(1, target.long().unsqueeze(1), 1)?  Can this self.targets_classes be understood as the one-hot labels? 

Thanks.

**Comment by lnsmith54** [comment]
As I recall, this line of code was in the code from the asymmetric loss (see https://github.com/Alibaba-MIIL/ASL) and it does represent the target class labels.

**Comment by whuhxb** [comment]
@lnsmith54 
OK. Thanks. 
Could this be also represented as self.targets_classes = tf.one_hot(tf.expand_dims(target,1), num_classes), right?
